HEN_HOUSE/scripts/configure.expect

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/expect -f
+
+set timeout -1
+
+# configuration filename
+set confname "expect.conf"
+if {$argc > 0} {
+    set confname [lindex $argv 0]
+}
+
+# clear environment variables
+set env(HEN_HOUSE)  ""
+set env(EGS_CONFIG) ""
+set env(EGS_HOME)   ""
+
+# spawn EGSnrc configuration script
+spawn HEN_HOUSE/scripts/configure
+
+# Information
+expect "Press enter to continue."                       { send "\n" }
+
+# Fortran compiler
+expect "Input fortran compiler:"                        { send "\n" }
+expect "Input standard compilation flags:"              { send "\n" }
+expect "Input optimization flags:"                      { send "\n" }
+expect "Input flags for debugging:"                     { send "\n" }
+expect "Input libraries that your compiler may need:"   { send "\n" }
+
+# C compiler
+expect "Input the one you would like to use:"           { send "\n" }
+expect "Input C compiler flags to use:"                 { send "\n" }
+
+# Configuration
+expect "Input path to EGSnrc HEN_HOUSE directory:"      { send "\n" }
+expect "Choose a configuration file name:"              { send "$confname\n" }
+set timeout 1
+expect "Do you want to overwrite ?"                     { puts "no\n"; exit }
```

Comment:
I'd rather not overwrite **anything** in an automated script. If there is an overwrite condition, then the user should take care of that manually. The default conf name here is `expect.conf`, which is unlikely to collide with something user selected, but who knows? Since this expect script is in the repository, someone might use it to configure without having to answer all the questions...